exporters/otlp/src/recordable.cc

```diff
@@ -111,21 +111,36 @@ void Recordable::SetAttribute(nostd::string_view key,
   }
 }
 
+void Recordable::SetAttribute(nostd::string_view key,
+                              const opentelemetry::common::AttributeValue &value) noexcept
+{
+  auto *attribute = span_.add_attributes();
+  PopulateAttribute(attribute, key, value);
+}
+
 void Recordable::AddEvent(nostd::string_view name,
                           core::SystemTimestamp timestamp,
                           const trace::KeyValueIterable &attributes) noexcept
 {
   auto *event = span_.add_events();
   event->set_name(name.data(), name.size());
   event->set_time_unix_nano(timestamp.time_since_epoch().count());
-  // TODO: handle attributes
+
+  attributes.ForEachKeyValue([&](nostd::string_view key, common::AttributeValue value) noexcept {
+    PopulateAttribute(event->add_attributes(), key, value);
+    return true;
+  });
 }
 
 void Recordable::AddLink(opentelemetry::trace::SpanContext span_context,
                          const trace::KeyValueIterable &attributes) noexcept
 {
-  (void)span_context;
-  (void)attributes;
+  auto *link = span_.add_links();
+  attributes.ForEachKeyValue([&](nostd::string_view key, common::AttributeValue value) noexcept {
+    PopulateAttribute(link->add_attributes(), key, value);
+    return true;
+  });
+  // TODO: Populate trace_id, span_id and trace_state when these are supported by SpanContext
```

Comment:
Can you please submit an issue for this, similar to [this one](https://github.com/open-telemetry/opentelemetry-cpp/issues/201)?

Reply:
Done! https://github.com/open-telemetry/opentelemetry-cpp/issues/214